docs/developer/cli-arguments.md

```diff
@@ -46,6 +46,7 @@ Flags:
       --custom-resource-state-config-file string   Path to a Custom Resource State Metrics config file (experimental)
       --custom-resource-state-only                 Only provide Custom Resource State metrics (experimental)
       --enable-gzip-encoding                       Gzip responses when requested by clients via 'Accept-Encoding: gzip' header.
+      --enable-no-node-scrape                      This configuration is used in conjunction with node configuration. When this configuration is true, node configuration is empty and the metric of no scheduled pods is scraped. This is experimental.
```

Comment:
Suggestion: Move this closer to the `--node` argument and rename from no_node to unscheduled_pods or similar?

```suggestion
      --node-sharding-unscheduled-pods-only                     This configuration is used in conjunction with node configuration. When this configuration is true, node configuration is empty and the metric of unscheduled pods is scraped from the Kubernetes API. This is experimental.
```